API.md

```diff
@@ -23,9 +23,13 @@ For the endpoints that return the details of a case:
 
 &mdash;an optional `iac` boolean query parameter can be used to specify that the JSON response include the unique access code associated with the case. For example:
 
-* `GET /cases/partyid/{partyId}?iac=true`
+* `GET /cases/partyid/{partyId}?iac=true?max_cases_per_survey=<n>`
```

Comment:
I would refactor this to talk about each optional parameter separately to better fit the sentence above. I'd also like to see the parameterised bit written as `{n}` rather than `<n>` for consistency

Reply:
Good catch , changing